app/main.py

```diff
@@ -1 +1,52 @@
-# write your code here
+class SoftwareEngineer:
+    def __init__(self, name: str) -> None:
+        self.name = name
+        self.skills = []
+
+    def learn_skill(self, skill: str) -> None:
+        self.skills.append(skill)
+
+
+class FrontendDeveloper(SoftwareEngineer):
+    def __init__(self, name: str) -> None:
+        super().__init__(name=name)
+        self.skills += ["JavaScript",
+                        "HTML",
+                        "CSS"]
```

Comment:
As for me, this style for me is not ideal for separating values into the new line.
Here you can use 1 line `["JavaScript", "HTML", "CSS"]`.
Or separate like `black` suggest (but only, if more values than here):
```python
skills = [
    "First skill",
    "Second skill",
    "Third skill",
    "Fourth skill",
    "Fifth skill",
]
```